src/cdk/a11y/focus-trap/focus-trap.ts

```diff
@@ -333,15 +333,10 @@ export class FocusTrap {
 /** Factory that allows easy instantiation of focus traps. */
 @Injectable({providedIn: 'root'})
 export class FocusTrapFactory {
-  private _document: Document;
-
   constructor(
       private _checker: InteractivityChecker,
       private _ngZone: NgZone,
-      @Inject(DOCUMENT) _document: any) {
-
-    this._document = _document;
-  }
+      @Inject(DOCUMENT) private _document: any) {}
```

Comment:
The previous approach was on purpose, because it allowed us to keep the strong typing on the `_document`.